cmd/cli/plugin/package/go.mod

```diff
@@ -0,0 +1,248 @@
+module github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework => ../../../../
+	github.com/vmware-tanzu/tanzu-framework/apis/cli => ../../../../apis/cli
+	github.com/vmware-tanzu/tanzu-framework/apis/config => ../../../../apis/config
+	github.com/vmware-tanzu/tanzu-framework/apis/run => ../../../../apis/run
+	github.com/vmware-tanzu/tanzu-framework/cli/core => ../../../../cli/core
+	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../../../cli/runtime
+	github.com/vmware-tanzu/tanzu-framework/packageclients => ../../../../packageclients
+	github.com/vmware-tanzu/tanzu-framework/tkg => ../../../../tkg
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.5
+	sigs.k8s.io/kind => sigs.k8s.io/kind v0.11.1
+)
+
+require (
+	github.com/aunum/log v0.0.0-20200821225356-38d2e2c8b489
+	github.com/cppforlife/go-cli-ui v0.0.0-20200716203538-1e47f820817f
+	github.com/k14s/semver/v4 v4.0.1-0.20210701191048-266d47ac6115
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.20.2
+	github.com/pkg/errors v0.9.1
+	github.com/spf13/cobra v1.5.0
+	github.com/stretchr/testify v1.8.0
+	github.com/vmware-tanzu/carvel-kapp-controller v0.35.0
+	github.com/vmware-tanzu/carvel-kapp-controller/cli v0.0.0-20220617205641-1d376ccc7312
+	github.com/vmware-tanzu/tanzu-framework v0.26.0
```

Comment:
Question: Should we use ```v0.0.0-00010101000000-000000000000``` here?

Reply:
+1

Reply:
I would say there shouldn't be dependency on tanzu-framework

Reply:
> Question: Shouldn't we use v0.0.0-00010101000000-000000000000 here?
Done.